### PR TITLE
Clean up constants based on file/dir

### PR DESCRIPTION
**Pull request by ZeeshanM96**
### Description:
Constants in the constants.py should be named after what they represent e.g. `_DIR` for directories and `_FILE` for files.

### Testing Screenshots:
jujustatus

logs

image

### CODE REVIEW

**Comment by awnns** [comment]
I believe the USER constant was supposed to remain. There is a separate constant for the user's home directory already.

**Comment by awnns** [comment]
Also probably, we should use names like "HOME_DIR" instead of "HOME_PATH_DIR" in the interest of avoiding https://en.wikipedia.org/wiki/Tautology_(language).

**Comment by ZeeshanM96** [comment]
@awnns I tried looking up for `USER_DIR`, and couldn't find any. All of the `USER_DIR` are coming from the `constant.py` file. Please correct me if I am wrong.

Screenshot from 2024-05-10 10-22-37

**Comment by awnns** [comment]
> @awnns I tried looking up for `USER_DIR`, and couldn't find any. All of the `USER_DIR` are coming from the `constant.py` file. Please correct me if I am wrong.
> 
> Screenshot from 2024-05-10 10-22-37

image

**Comment by ZeeshanM96** [comment]
@jonathanudd @awnns i have made the naming convention better. Please have a look and let me know if you have any remarks

File: src/constants.py [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
-USER = 'polkadot'
-SERVICE_NAME = USER
-HOME_PATH = Path('/home/polkadot')
-BINARY_PATH = Path(HOME_PATH, 'polkadot')
-CHAIN_SPEC_PATH = Path(HOME_PATH, 'spec')
-NODE_KEY_PATH = Path(HOME_PATH, 'node-key')
-DB_CHAIN_PATH = Path(HOME_PATH, '.local/share/polkadot/chains')
-DB_RELAY_PATH = Path(HOME_PATH, '.local/share/polkadot/polkadot')
-WASM_PATH = Path(HOME_PATH, 'wasm')
+USER_DIR = 'polkadot'
```
[the remark this answers is not in the document]
Comment (ZeeshanM96) [reply src/constants.py]: Yes, I have updated the changes and ran the charm normally and also with service arguments to see the service getting restarted and log that's `config-changed`. I have attached screenshots in the descriptions as well.
e.g: `unit-polkadot-0: 11:45:12 INFO juju.worker.uniter.the operation ran "config-changed" hook (via hook dispatching script: dispatch)`

**Review by jakobilobi — APPROVED** [review]

**Review by awnns — CHANGES_REQUESTED** [review]
You have not ensured that all uses of the constants have had their names changed to the name in the new constants file. The version you proposed would break the charm and cause numerous functions to throw exceptions.

**Pull request merged**